edx_name_affirmation/tests/test_models.py

```diff
@@ -25,15 +32,24 @@ def setUp(self):
         )
         return super().setUp()
 
-    def test_histories(self):
+    @patch('edx_name_affirmation.models.SoftwareSecurePhotoVerification')
+    def test_histories(self, sspv_mock):
```

Comment:
This test looks good in how it tests `verification_attempt_status`, but I'd prefer it in a separate test specific to that property as opposed to including it in the test for the history.

Reply:
Moved the `verification_attempt_status` to its own test.